src/V3File.cpp

```diff
@@ -15,10 +15,11 @@
 //*************************************************************************
 
 #include "config_build.h"
-#include "verilatedos.h"
 
 #include "V3File.h"
 
+#include "verilated.h"
```

Comment:
Please keep the verilatedos.h include.

I don't want verilated included in Verilator itself, if necessary we need to move some functions out of it to a new include header.    Or can you use verilated_threads.h?

Reply:
P.S. If verilated refactoring verilated.h is needed, that might be better in another separate preliminary pull.

Reply:
I've used `` verilated_threads.h`` instead.